beacon_node/store/src/metadata.rs

```diff
@@ -120,14 +120,10 @@ impl StoreItem for AnchorInfo {
 }
 
 /// Database parameters relevant to blob sync.
-#[derive(Debug, PartialEq, Eq, Clone, Encode, Decode, Serialize, Deserialize)]
+#[derive(Debug, PartialEq, Eq, Clone, Encode, Decode, Serialize, Deserialize, Default)]
 pub struct BlobInfo {
-    /// The block root of the next blob that needs to be added to fill in the history.
-    pub oldest_blob_parent: Hash256,
     /// The slot before which blobs are available.
     pub oldest_blob_slot: Slot,
```

Comment:
I have defaulted to using "of a higher epoch number" = "younger" and "of a lower epoch number" = "older". Older and younger makes sense because "parent hash" is deeply imprinted in our brain already.